sqlconnect/internal/mysql/config_test.go

```diff
@@ -0,0 +1,51 @@
+package mysql_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/rudderlabs/sqlconnect-go/sqlconnect/internal/mysql"
+)
+
+func TestConfig(t *testing.T) {
```

Comment:
minor: Do you think we should add test for ConnectionString? Its only string format, so not very useful.

Reply:
it is tested (indirectly) through the integration test